GameServer/Logic/Logger.cs

```diff
@@ -72,16 +72,19 @@ private class LogWriter
         {
             public string _executingDirectory;
             public string _logFileName;
+            private bool _fileCreated = false;
             private object _locker = new object();
 
-            public LogWriter(string executingDirectory, string logFileName)
+            public LogWriter(string executingDirectory)
```

Comment:
No need to remove the log name parameter. You're turning something dynamic into hard coded, which is a bad idea.